### PR TITLE
Switch from mysql-async to ghmattimysql

### PR DESCRIPTION
**Pull request by GoatG33k**
- Drop mysql-async, switch to ghmattimysql
- Performance improvements
- Code cleanup

### CODE REVIEW

**Comment by RockySouthpaw** [comment]
Will test sometime tomorrow (like 12 hours ish.) :)

File: server/sv_events.lua [hunk 1]
```diff
@@ -53,40 +55,38 @@ RegisterNetEvent('NGWD:storeVehicle', function(vehicle, garageName, plate, model
...
+                        execute('UPDATE ngwd_vehicles SET garage = @garage, vehicleProperties = @vehicleProperties, vehicleCondition = @vehicleCondition, vehicleMods = @vehicleMods WHERE plate = @plate', { 
+                            ['owner']                  = identifier, 
+                            ['modelHash']              = modelHash,
+                            ['localizedName']          = localizedName,
+                            ['plate']                  = plate, 
+                            ['garage']                 = garageName,
+                            ['vehicleProperties']      = json.encode(vehicleProperties),
+                            ['vehicleCondition']       = json.encode(vehicleCondition),
+                            ['vehicleMods']            = json.encode(vehicleMods)
+                        }, function() end)
```
Comment (TasoOneAsia) [inline server/sv_events.lua:73]: Presuming @RockySouthpaw doesn't need to do anything with affected rows, we can simply omit the cb

File: server/sv_events.lua [hunk 2]
```diff
@@ -53,40 +55,38 @@ RegisterNetEvent('NGWD:storeVehicle', function(vehicle, garageName, plate, model
...
+                            execute('UPDATE ngwd_vehicles SET garage = @garage, vehicleProperties = @vehicleProperties, vehicleCondition = @vehicleCondition, vehicleMods = @vehicleMods WHERE plate = @plate', { 
+                                ['owner']                  = identifier, 
+                                ['modelHash']              = modelHash,
+                                ['localizedName']          = localizedName,
+                                ['plate']                  = plate, 
+                                ['garage']                 = garageName,
+                                ['vehicleProperties']      = json.encode(vehicleProperties),
+                                ['vehicleCondition']       = json.encode(vehicleCondition),
+                                ['vehicleMods']            = json.encode(vehicleMods)
+                            }, function() end)
```
Comment (TasoOneAsia) [inline server/sv_events.lua:89]: Same as above

**Review by TasoOneAsia — APPROVED** [review]
Aside from the empty cb's that @RockySouthpaw might want to do something with LGTM.

File: server/sv_events.lua (hunk 1 above)
Comment (EvanDarwin) [reply server/sv_events.lua:73]: I will fix this as well as the `['key']` format like you requested in Discord

Comment (GoatG33k) [reply server/sv_events.lua:73]: I'll make this change, as well as fixing the `['key']` format as requested tonight

**Comment by RockySouthpaw** [comment]
Can you also update the read me to include `ghmattimysql` as a dependency?

**Comment by RockySouthpaw** [comment]
There's some SQL errors when doing `/buy`. Most likely related to my msql version though 🤷🏻 

image

If @TasoOneAsia can test later that would be helpful (just go to legion at `x = 228.40103149414, y = -811.4248046875, z = 30.479082107544` and do `/buy` while ine a vehicle and `/spawn`.

**Pull request merged**